airflow/include/exampleplugin.go

```diff
@@ -5,42 +5,65 @@ import "strings"
 // ExamplePlugin created with astro airflow init
 var ExamplePlugin = strings.TrimSpace(`
 from airflow.plugins_manager import AirflowPlugin
-from flask_admin.base import MenuLink
+from airflow.version import version
```

Comment:
hm... if user still using airflow prior 2.0 and new astro cli?
may be
```suggestion
try:
    from flask_admin.base import MenuLink
except ImportError:
    from airflow.version import version
```
cc @ashb